### PR TITLE
New README.md

### PR DESCRIPTION
**Pull request by rebeccahum**
As per #261, we have a `readme.txt` which is not aesthetically pleasing.  I have used the contents from that file and created a `README.md` with Markdown formatting.

### CODE REVIEW

**Comment by trepmal** [comment]
We won't want to remove the readme.txt entirely, as that's still needed for the plugin hosted on WordPress.org. https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/ (there's a validator there in case things need to be tweaked/updated)

File: README.md [hunk 1]
```diff
@@ -17,68 +18,68 @@ On the frontend, use the [Co-Authors Plus template tags](http://vip.wordpress.co
...
+## Frequently Asked Questions 
 
-= How do I add Co-Authors Plus support to my theme? =
+* How do I add Co-Authors Plus support to my theme?
 
-If you've just installed Co-Authors Plus, you might notice that the bylines are being added in the backend but aren't appearing on the frontend. You'll need to [add the template tags to your theme](http://vip.wordpress.com/documentation/incorporate-co-authors-plus-template-tags-into-your-theme/) before the bylines will appear.
+> If you've just installed Co-Authors Plus, you might notice that the bylines are being added in the backend but aren't appearing on the frontend. You'll need to [add the template tags to your theme](http://vip.wordpress.com/documentation/incorporate-co-authors-plus-template-tags-into-your-theme/) before the bylines will appear.
 
-= What happens to posts and pages when I delete a user assigned to a post or page as a coauthor? =
+* What happens to posts and pages when I delete a user assigned to a post or page as a coauthor? =
```
Comment (trepmal) [inline README.md]: stray `=` at the end

File: README.md [hunk 2]
```diff
@@ -17,68 +18,68 @@ On the frontend, use the [Co-Authors Plus template tags](http://vip.wordpress.co
...
 This plugin is an almost complete rewrite of the Co-Authors plugin originally developed at [Shepherd Interactive](http://www.shepherd-interactive.com/) (2007). The original plugin was inspired by the 'Multiple Authors' plugin by Mark Jaquith (2005).
 
-== Frequently Asked Questions ==
+## Frequently Asked Questions 
 
-= How do I add Co-Authors Plus support to my theme? =
+* How do I add Co-Authors Plus support to my theme?
 
-If you've just installed Co-Authors Plus, you might notice that the bylines are being added in the backend but aren't appearing on the frontend. You'll need to [add the template tags to your theme](http://vip.wordpress.com/documentation/incorporate-co-authors-plus-template-tags-into-your-theme/) before the bylines will appear.
+> If you've just installed Co-Authors Plus, you might notice that the bylines are being added in the backend but aren't appearing on the frontend. You'll need to [add the template tags to your theme](http://vip.wordpress.com/documentation/incorporate-co-authors-plus-template-tags-into-your-theme/) before the bylines will appear.
```
Comment (trepmal) [inline README.md]: I'm not totally sold on the `>`/quote indenting here. I think a regular paragraph would be fine

File: README.md [hunk 3]
```diff
@@ -17,68 +18,68 @@ On the frontend, use the [Co-Authors Plus template tags](http://vip.wordpress.co
...
 
-= What happens to posts and pages when I delete a user assigned to a post or page as a coauthor? =
+* What happens to posts and pages when I delete a user assigned to a post or page as a coauthor? =
 
-When a user is deleted from WordPress, they will be removed from all posts for which they are co-authors. If you chose to reassign their posts to another user, that user will be set as the coauthor instead.
+> When a user is deleted from WordPress, they will be removed from all posts for which they are co-authors. If you chose to reassign their posts to another user, that user will be set as the coauthor instead.
 
-= Can I use Co-Authors Plus with WordPress multisite? =
+* Can I use Co-Authors Plus with WordPress multisite? =
```
Comment (trepmal) [inline README.md]: another stray `=`

File: README.md [hunk 4]
```diff
@@ -17,68 +18,68 @@ On the frontend, use the [Co-Authors Plus template tags](http://vip.wordpress.co
...
+> To create new guest author profiles, a WordPress will need the `list_users` capability. This is typically granted to the Administrator role, but can be altered with the `coauthors_guest_author_manage_cap` filter.
 
-= Can I easily create a list of all co-authors? =
+* Can I easily create a list of all co-authors? 
 
-Yep! There's a template tag called `coauthors_wp_list_authors()` that accepts many of the same arguments as `wp_list_authors()`. Look in template-tags.php for more details.
+> Yep! There's a template tag called `coauthors_wp_list_authors()` that accepts many of the same arguments as `wp_list_authors()`. Look in template-tags.php for more details.
 
-== Upgrade Notice ==
+## Upgrade Notice
```
Comment (trepmal) [inline README.md]: I the context of the md file for Github, we can lose the "upgrade notice" section. That's specific to how WordPress displays pending updates within WordPress.

**Comment by rebeccahum** [comment]
@trepmal Added the readme.txt back, removed stray =s, removed quotes and removed "Update notice" section.  Let me know if there is anything else that needs to be changed, thanks!

**Pull request merged**